jwst/tests_nightly/general/nirspec/test_nrs1_ifu_wcs.py

```diff
@@ -0,0 +1,53 @@
+import os
+import pytest
+import numpy as np
+from numpy.testing import utils
+
+from jwst.assign_wcs import AssignWcsStep, nirspec
+
+from jwst.datamodels import ImageModel
+
+from ..helpers import add_suffix
+
+pytestmark = [
+    pytest.mark.usefixtures('_jail'),
+    pytest.mark.skipif(not pytest.config.getoption('bigdata'),
+                       reason='requires --bigdata')
+]
+
+
+def test_nirspec_nrs1_wcs(_bigdata):
+    """
+
+    Regression test of creating a WCS object and doing pixel to sky transformation.
+
+    """
+    output_file_base, output_file = add_suffix('nrs1_ifu_wcs_output.fits', 'assignwcsstep')
+
+    try:
+        os.remove(output_file)
+    except:
+        pass
+
+    input_file = os.path.join(_bigdata, 'nirspec', 'test_wcs', 'nrs1-ifu', 'jw00011001001_01120_00001_NRS1_rate.fits')
```

Comment:
Here's a nice example of `os.path.join` usage I mentioned above. Kudos to this one. :clap: